hack/lib/util.sh

```diff
@@ -729,6 +729,16 @@ function kube::util::ensure_dockerized {
   fi
 }
 
+# kube::util::ensure-bash-version
+# Check if we are using a supported bash version
+#
+function kube::util::ensure-bash-version {
+  if ((${BASH_VERSINFO[0]}<5)); then
+    echo "ERROR: This script needs at bash version at least 5.x"
```

Comment:
```suggestion
    echo "ERROR: This script requires a minimum bash version of 5.0"
```

Reply:
Obviously bash 5.x can work normally, but this further narrows the optional range of compilation environment (eg. CentOS7 Ubuntu16.04), maybe bash 4.2 is the recommended value, better compatibility should also be one of the goals of the community.

Reply:
@JoshuaAndrew this PR reflects reality, please feel free to follow up with changes to make things work in bash 4.2 

thanks!